apps/ejabberd/priv/mysql.sql

```diff
@@ -212,7 +212,8 @@ CREATE TABLE mam_message(
   search_body text,
   PRIMARY KEY (user_id, id),
   INDEX i_mam_message_rem USING BTREE (user_id, remote_bare_jid, id)
-)  ENGINE=InnoDB
+)  CHARACTER SET utf8
```

Comment:
Should we instead use [`utf8mb4`](https://stackoverflow.com/questions/30074492/what-is-the-difference-between-utf8mb4-and-utf8-charsets-in-mysql)?

Reply:
It is a separate task that will involve converting all tables to utf8mb4 and providing a script to perform migration. It will also probably require changing indices' sizes.

Reply:
Ok, I can see that other tables are `utf8` already.